native/src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -77,25 +78,32 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
 
   // Check whether another page in the same city is loading, e.g. because it is being refreshed.
   // This is important for displaying the loading spinner.
-  const otherPageLoading = values<CategoryRouteStateType>(state.categoriesRouteMapping)
-    .filter(route => city === route.city && path !== route.path && language === route.language)
+  const otherPageLoading = values<RouteStateType>(state.routeMapping)
+    .filter(route => route.routeType === 'category' &&
+      city === route.city &&
+      path !== route.path &&
+      language === route.language)
     .some(route => route.status === 'loading')
 
-  const newCategoriesRouteMapping = { ...state.categoriesRouteMapping }
+  const newRouteMapping = { ...state.routeMapping }
 
   if (refresh) {
     // Update all open routes in the same city with the new content in case the content has been refreshed
-    entries<CategoryRouteStateType>(state.categoriesRouteMapping)
-      .filter(([_, route]) => city === route.city && path !== route.path && language === route.language)
+    entries<RouteStateType>(state.routeMapping)
+      .filter(([_, route]) => route.routeType === 'category' &&
+        city === route.city &&
+        path !== route.path &&
+        language === route.language)
       .forEach(([key, route]) => {
-        if (route.status === 'languageNotAvailable') {
+        if (route.routeType !== 'category' || route.status === 'languageNotAvailable') {
```

Comment:
`route.routeType !== 'category'` necessary for typing. However I am open for ideas to omit this here.

Reply:
Probably by typing the parameters or the return values?  But not sure about that. If this is necessary then it is necessary, I think it is fine.

Reply:
As it is not trivial changing the type in filter i will keep it.

Reply:
You could use the constant instead of the string though :)